tests/test_geomopt_cli.py

```diff
@@ -217,15 +192,7 @@ def test_vectors_not_fully_opt(tmp_path):
     )
     assert result.exit_code == 0
 
-    # Read log file
-    with open(log_path, encoding="utf8") as log_file:
-        logs = yaml.safe_load(log_file)
-
-    # Check for correct messages anywhere in logs
-    messages = ""
-    for log in logs:
-        messages += log["message"]
-    assert "hydrostatic_strain: True" in messages
+    check_log_contents(log_path, includes=["Using filter", "hydrostatic_strain: True"])
```

Comment:
Looks good! Though we might want to change it to `assert_log_contains`?

Reply:
Sounds good to me, thanks